frontends/election-manager/src/screens/logic_and_accuracy_screen.tsx

```diff
@@ -13,25 +13,43 @@ export function LogicAndAccuracyScreen(): JSX.Element {
   return (
     <NavigationScreen>
       <Prose maxWidth={false}>
-        <h1>L&amp;A Materials</h1>
+        <h1>L&A Testing Documents</h1>
         {isLiveMode ? (
           <p>
-            L&amp;A materials are not available after live CVRs have been
-            imported.
+            L&A materials are not available after live CVRs have been imported.
           </p>
         ) : (
           <React.Fragment>
             <p>
-              Go to the “Tally” tab and print the “Unofficial Full Election
-              Tally Report”. This report is referred to as the “Zero Report”
-              because — before CVRs have been imported — all tallies should be
-              zero.
+              Print the following reports and ballots in preparation for L&A
+              Testing.
```

Comment:
We're capitalizing "L&A Testing" here but not below when we say "Print this tally report to document that no votes have been tallied before L&A testing begins."

Let's stick with one for consistency (not including titles of course). I lean toward lower case "testing"

Reply:
lowercased.